Cargo.toml

```diff
@@ -61,6 +61,9 @@ turbopack = [
   "path:crates/node-file-trace",
 ]
 
+[workspace.lints.clippy]
+too_many_arguments = "allow"
```

Comment:
Do we have a lot of violations of this? Seems like a stylistic thing so fine to disable this rule, but curious if it should be localized or not

Reply:
The violations are quite localized across 3 turbopack crates:

crates/turbopack-core -- 2
crates/turbopack-ecmascript -- 8
crates/turbopack-node -- 24

However it might be surprising for this only to be allowed in these crates alone rather than anywhere in the workspace.